model-optimizer/mo/front/tf/extractors/utils.py

```diff
@@ -72,7 +72,10 @@ def tf_tensor_content(tf_dtype, shape, pb_tensor):
         value = np.array(np.frombuffer(pb_tensor.tensor_content, type_helper[0]))
     else:
         # load typed value
-        value = np.array(type_helper[1](pb_tensor), dtype=type_helper[0])
+        try:
+            value = np.array(type_helper[1](pb_tensor), dtype=type_helper[0])
+        except UnicodeDecodeError:
```

Comment:
Does this exception happen for real (unexpected) issue in some case? Possibly we need some warning here, but I am not sure about it.

Reply:
We should check for the data type here. And have a try only for strings I think.

Reply:
And for sure we should emit warning message for the user.

Reply:
@lazarevevgeny What kind of the message should be? Something like "Possible problem decoding a byte-represented string value"? I do not quite understand how to describe this.

Reply:
`log.error('Failed to parse a tensor with Unicode characters. Note that Inference Engine does not support string literals, so the string constant should be eliminated from the graph.', extra={'is_warning': True})`